weldx/tests/test_utility.py

```diff
@@ -400,13 +400,8 @@ def test_compare_nested_raise(a, b):  # noqa: D102
         argvalues=[
             ((1, 2, 3), [1, 2, 3], True),
             ((1, 2, 3), [1, 2, 0], False),
-            ({1, 2, 3}, {3, 2, 1}, True),
-            ({1, 2, 3}, [3, 2, 1], False),
-            ({1, 2, 3}, {1, 2, 0}, False),
-            ({1, 2, 3}, [1, 2, 3], True),
-            ([1, 2, 3], {1, 2, 3}, True),
```

Comment:
Since sets are no longer considered a "nested" datastructure here `compare_nested` cannot be used on sets directly